### PR TITLE
Added and fixed some Hindi translations

### PR DESCRIPTION
**Pull request by dhruvmisra**
Added Hindi strings for Import and Export Users and fixed one more.

Before:
localeIssue

After:
localeFix

### CODE REVIEW

**Comment by llaske** [comment]
@tarunsinghal92 or @NikhilM98 could you review?

File: dashboard/public/l10n/locales.ini [hunk 1]
```diff
@@ -776,7 +776,7 @@ ClassroomUpdated=कक्षा {{name}} को सफलतापूर्व
 ClassroomCreated=कक्षा {{name}} सफलतापूर्वक बनाया गया है!
 ClassroomDeleted=कक्षा {{name}} को सफलतापूर्वक हटा दिया गया है!
 DoYouWantDeleteClassroom=क्या आप कक्षा {{classroom}} को हटाना चाहते हैं?
-ShortForBytes=ब
```
Comment (NikhilM98) [inline dashboard/public/l10n/locales.ini]: This should not be changed. The data is in Bytes not Bits.

File: dashboard/public/l10n/locales.ini [hunk 2]
```diff
@@ -817,8 +817,8 @@ HowManyUsersActiveLastWeek=पिछले सप्ताह में कि
 HowManyUsersActiveLastMonth=पिछले महीने में कितने उपयोगकर्ता सक्रिय हैं
 HowManyUsersActiveLastYear=पिछले वर्ष में कितने उपयोगकर्ता सक्रिय हैं
 CountStudents=छात्र गणना
-ImportUsers=Import users
-ExportUsers=Export users
+ImportUsers=उपयोगकर्ता आयात करें
```
Comment (NikhilM98) [inline dashboard/public/l10n/locales.ini]:
```suggestion
ImportUsers=आयात उपयोगकर्ता
```
Shorter button text

File: dashboard/public/l10n/locales.ini [hunk 3]
```diff
@@ -817,8 +817,8 @@ HowManyUsersActiveLastWeek=पिछले सप्ताह में कि
...
+ExportUsers=उपयोगकर्ता निर्यात करें
```
Comment (NikhilM98) [inline dashboard/public/l10n/locales.ini]:
```suggestion
ExportUsers=निर्यात उपयोगकर्ता
```

File: dashboard/public/l10n/locales.ini (hunk 1 above)
Comment (dhruvmisra) [reply dashboard/public/l10n/locales.ini]: Thanks for the review. Although shouldn't this follow the common trend of using बी for Bytes? For example in ShortForKilobytes=केबी and ShortForMegabytes=एमबी.

**Comment by llaske** [comment]
@NikhilM98 could you check the fix?

File: dashboard/public/l10n/locales.ini (hunk 1 above)
Comment (NikhilM98) [reply dashboard/public/l10n/locales.ini]: If you use बी for Byte then it could cause confusion between Bit and Byte.
@ashish0910 what do you think?

Comment (ashish0910) [reply dashboard/public/l10n/locales.ini]: I think ब makes more sense .

Comment (dhruvmisra) [reply dashboard/public/l10n/locales.ini]: Fixed in latest commit.

**Comment by llaske** [comment]
Nice. Thanks.

**Pull request merged**